src/types/profile-derived.js

```diff
@@ -320,7 +320,6 @@ export type AccumulatedCounterSamples = {|
 export type CollectedCustomMarkerSamples = {|
   +minNumber: number,
   +maxNumber: number,
-  markers: Marker[],
```

Comment:
Optional, of we can do it in a follow-up: Thanks for removing the `markers` array. It could be also nice to rename the `indexes` array to `markerIndexes` to make it clearer. But leaving that up to you.